dbt/node_runners.py

```diff
@@ -368,7 +370,15 @@ def print_results_line(cls, results, execution_time):
 
     @classmethod
     def after_run(cls, config, adapter, results, manifest):
-        cls.safe_run_hooks(config, adapter, manifest, RunHookType.End)
+        # in on-run-end hooks, provide the value 'schemas', which is a list of
+        # unique schemas that successfully executed models were in
+        # errored failed skipped
+        schemas = list(set(
+            r.node.schema for r in results
+            if not any((r.errored, r.failed, r.skipped))
+        ))
+        cls.safe_run_hooks(config, adapter, manifest, RunHookType.End,
```

Comment:
would it be crazy to just add the `results` into `extra_content` here too? I imagine it would be useful for users to iterate over the results and do something intelligent with errors, for example. Any drawbacks to that idea?